showcase/src/styles/styles.scss

```diff
@@ -2,3 +2,4 @@
 @import "sidenav_temp";
 @import "~basscss/css/basscss.css";
 @import "~normalize.css/normalize.css";
+@import "~prismjs/themes/prism-okaidia.css";
```

Comment:
We should add documentation for this component and mention that we have to add an import to prismjs to make it work. Otherwise, it's not obvious...

Reply:
Thru, but I'm waiting for the Showcase update, so that we can start adding pages for each component,